src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1213,9 +1213,12 @@ static bool addTriggeredSubscriptions_withCache
     {
```

Comment:
Not sure...

Is this PR related with the fix to https://github.com/telefonicaid/fiware-orion/issues/1406?